docs/installation.md

```diff
@@ -54,4 +54,4 @@ What you need to do to (re)start the project:
   make frontend/start
   ```
 
-  The project should now be running at [localhost:8000](http://localhost:8000).
+  The project should now be running at [localhost:8000](http://localhost:8000). The fixtures load has created an account you can access by using "admin" as user name and password.
```

Comment:
you can explicit even more the credentials: 

```
an account you can access with the following credentials: 
- username: `admin`
- password: `admin`
```

I also think that the project is running at `localhost:3000` since we moved to Webpack Dev Server as the frontend webserver.

You may also mention that the Django Admin interface is running at `localhost:8000/admin/` and that the trailing slash is important.